sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -983,19 +983,17 @@ public partial class HttpClientTransport : Azure.Core.Pipeline.HttpPipelineTrans
         public override void Process(Azure.Core.HttpMessage message) { }
         public override System.Threading.Tasks.ValueTask ProcessAsync(Azure.Core.HttpMessage message) { throw null; }
     }
-    public partial class HttpPipeline : System.Net.ClientModel.Core.Pipeline<Azure.Core.HttpMessage>
+    public partial class HttpPipeline
```

Comment:
Note that with this change, there is now no requirement coming from the type system that HttpMessage inherit from PipelineMessage, so we could feasibly remove that inheritance structure.  This motivates the question ... what value do we get from HttpMessage inheriting from PipelineMessage?  Initial thoughts on this include:
- Enforces the contract similarity between *message* types
- If any complex technical implementation lives in PipelineMessage, this would not be duplicated
- If we wanted substitutability in client code for some reason of PipelineMessage and HttpMessage, we wouldn't get it without this inheritance relationship
- Depending on our needs from the Options type abstractions, we might need substitutability/contract enforcement here